data/serializer/src/main/java/tech/pegasys/teku/api/response/v1/node/PeerScore.java

```diff
@@ -0,0 +1,49 @@
+package tech.pegasys.teku.api.response.v1.node;
```

Comment:
I'd also move this to: tech.pegasys.teku.api.response.v1.teku

Reply:
I've moved this also. Thank you!